apps/site/styles/index.css

```
@@ -6,10 +6,217 @@
+  --color-accent2-300: #fbb4d2;
+  --color-accent2-400: #f68bb7;
+  --color-accent2-500: #ed5393;
+  --color-accent2-600: #d6246e;
+  --color-accent2-700: #b01356;
+  --color-accent2-800: #8b1245;
+  --color-accent2-900: #411526;
+  --color-pulse-100: #0000330f;
+  --color-pulse-200: #00002d17;
+  --color-pulse-300: #ddeaf814;
+  --color-pulse-400: #d3edf81d;
+  --color-white: #fff;
+  --color-transparent: 'transparent';
+  --color-shadow: #101828;
+  --inherit: 'inherit';
+  --shadow-xs: 0px 1px 2px 0px var(--color-shadow, rgba(0, 0, 0 / 5%));
+  --shadow-lg:
+    0px 4px 6px -2px var(--color-shadow, rgba(0, 0, 0 / 3%)),
+    0px 12px 16px -4px var(--color-shadow, rgba(0, 0, 0 / 8%));
+  --font-open-sans: var(--font-open-sans);
+  --font-ibm-plex-mono: var(--font-ibm-plex-mono);
+  --font-weight-regular: 400;
+  --font-weight-medium: 500;
+  --font-weight-semibold: 600;
+  --font-weight-bold: 700;
+  --font-xs: 0.75rem;
+  --font-xs-line-height: 1rem;
+  --font-sm: 0.87rem;
+  --font-sm-line-height: 1.25rem;
+  --font-base: 1rem;
+  --font-base-line-height: 1.5rem;
+  --font-lg: 1.125rem;
+  --font-lg-line-height: 1.75rem;
+  --font-xl: 1.25rem;
+  --font-xl-line-height: 1.875rem;
+  --font-2xl: 1.5rem;
+  --font-2xl-line-height: 2rem;
+  --font-3xl: 1.875rem;
+  --font-3xl-line-height: 2.25rem;
+  --font-4xl: 2.25rem;
+  --font-4xl-line-height: 2.5rem;
+  --font-5xl: 3rem;
+  --font-5xl-line-height: 3rem;
+  --font-6xl: 3.75rem;
+  --font-6xl-line-height: 3.75rem;
+  --font-7xl: 4.5rem;
+  --font-7xl-line-height: 4.5rem;
+  --breakpoint-xs: 670px;
+  --animate-surf: surf 1s infinite ease-in-out;
+  --animate-pulse: pulse 500ms infinite alternate-reverse;
+  --animate-pulse-dark: pulse-dark 500ms infinite alternate-reverse;
+
+  @keyframes surf {
+    0% {
+      transform: translate(0, 0);
+    }
+
+    25% {
+      transform: translate(0, 6px);
+    }
+
+    50% {
+      transform: translate(0, -6px);
+    }
+
+    75% {
+      transform: translate(0, 3px);
+    }
+
+    100% {
+      transform: translate(0, 0);
+    }
+  }
+
+  @keyframes pulse {
+    from {
+      background: var(--color-pulse-100);
+    }
+
+    to {
+      background: var(--color-pulse-200);
+    }
+  }
+
+  @keyframes pulse-dark {
+    from {
+      background: var(--color-pulse-300);
+    }
+
+    to {
+      background: var(--color-pulse-400);
+    }
+  }
+}
+
+/*
+  The default border color has changed to `currentColor` in Tailwind CSS v4,
+  so we've added these compatibility styles to make sure everything still
+  looks the same as it did with Tailwind CSS v3.
+
+  If we ever want to remove these styles, we need to add an explicit border
+  color utility to any element that depends on these defaults.
+*/
+@layer base {
+  *,
+  ::after,
+  ::before,
+  ::backdrop,
+  ::file-selector-button {
+    border-color: var(--color-gray-200, currentColor);
+  }
+}
+
+@utility scrollbar-thin {
+  scrollbar-width: thin;
+}
+
+@utility max-w-8xl {
+  max-width: 95rem;
+}
+
```

I have created the example utility classes like this. It might be better to define them in a separate file

```suggestion
@utility bg-gradient-subtle-dark {
  background-image: linear-gradient(
    180deg,
    --alpha(var(--color-neutral-900) / 50%) 0%,
    --alpha(var(--color-neutral-900) / 0%) 48.32%
  );
}

@utility bg-gradient-subtle {
  background-image: linear-gradient(
    180deg,
    --alpha(var(--color-neutral-100) / 50%) 0%,
    --alpha(var(--color-neutral-100) / 0%) 48.32%
  );
}

@utility bg-gradient-subtle-gray {
  background-image: linear-gradient(
    180deg,
    var(--color-neutral-900) 0%,
    --alpha(var(--color-neutral-900) / 80%) 100%
  );
}

@utility bg-gradient-subtle-white {
  background-image: linear-gradient(
    180deg,
    var(--color-white) 0%,
    --alpha(var(--color-white) / 80%) 100%
  );
}

@utility bg-gradient-glow-backdrop {
  background-image: radial-gradient(
    8em circle at calc(50%) 10px,
    var(--color-green-500),
    transparent 30%
  );
}

@utility bg-gradient-radial {
  background-image: radial-gradient(
    var(--tw-gradient-from),
    var(--tw-gradient-to)
  );
}
```

---

it's working, thanks